### DISCUSSION TITLE
Github pages

### DISCUSSION THREAD

**Discussion by Spirou4D** [ItsCubeTime/BlenderVersionSelector#4] [General]
Excuse me but your main's page don't show the pictures of explanations.... 

EDIT : Sorry I must wait the long downloading...

**Comment by ItsCubeTime** [comment]
Oh hi there, seems I didnt get this in my mailbox for some reason. Theres a link to a video demonstrating how it works, but essentially you configure Windows to always open with the version selector (currently called main.exe, you can rename it to something more meaningful if you like) and then the version selector reads the first few characters of the blend file which tells the software which Blender version the file was last saved in.

Then if you've entered the versions and install locations in your settings.json file the version selector will select the right version, or prompt you which version you would like to open it with else wise (you enter the version number, say 293 or 3.0.0 (only the numbers matter, so it doesnt matter if you include dots or write say "Blender 3.0.0").

The software runs through the terminal, though it gives you info when you need to enter anything and 90% of the time, you wont have to interact with it

**Comment by ItsCubeTime** [comment]
> Excuse me but your main's page don't show the pictures of explanations....
> 
> EDIT : Sorry I must wait the long downloading...

I added a new "how it works" screenie to the README.md 👍
